app/i18n/en/namespaces/global.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+const global = {
+  siteID: 'Open Webslides',
```

Comment:
Why is this called `siteID`? I'd just call it `title` or simply `openwebslides`.

Reply:
Because the logo and accompanying text in the top-left corner of a webpage are referred to as the 'siteID' by someone who wrote a book about that stuff (coincidentally recommended to me by a friend and colleague). And I don't like 'title'. But I could go for 'openwebslides'.